lib/AuthorizeNetAIM.php

```diff
@@ -269,12 +269,12 @@ public function addLineItem($item_id, $item_name, $item_description, $item_quant
         $line_item = "";
         $delimiter = "";
         foreach (func_get_args() as $key => $value) {
-            $line_item .= $delimiter . preg_replace('/|/', '_', $value);
+            $line_item .= $delimiter . preg_replace("/\|/", '_', $value);
```

Comment:
Use single quotes here so the `\` character is considered a regular expression escape character and not a string escape character.

Even better:

```php
$line_item .= $delimiter . str_replace('<|>', '_', $value);
```

The referenced PDF documentation in the original commit notes the delimiter sequence is `<|>` and not `|`. There is no reason to use a regular expression here when it's just doing a simple search/replace operation.